src/time-collection.ts

```diff
@@ -45,8 +47,9 @@ export const convertToTimeCollection = <TSchema extends Document>(
         } as any,
       }
     },
-    preReplace: () => {
-      throw Error('Does not support replace')
+    preReplace: (obj: WithoutId<TSchema>): WithoutId<WithTime<TSchema>> => {
```

Comment:
I added the input and output types here to be consistent with this file, but none of these are required as they are inferred from the `Converter` type definition:
https://github.com/tianhuil/ts-mongo/blob/aacb20f11e146eb202277e1bd4ee6578d7be8003/src/converter.ts#L19-L39